src/constants/chainInfo.ts

```diff
@@ -2,8 +2,7 @@ import celoCircleLogoUrl from 'assets/images/celoCircle.png'
 import ethereumLogoUrl from 'assets/images/ethereum-logo.png'
 import optimismCircleLogoUrl from 'assets/images/optimismCircle.png'
 import polygonCircleLogoUrl from 'assets/images/polygonCircle.png'
-import arbitrumLogoUrl from 'assets/svg/arbitrum_logo.svg'
-import arbitrumCircleLogoUrl from 'assets/svg/arbitrum_logo.svg'
+import { default as arbitrumCircleLogoUrl, default as arbitrumLogoUrl } from 'assets/svg/arbitrum_logo.svg'
```

Comment:
- why the diff here?
- why is the same thing imported twice as two different variable names?

Reply:
I made an edit and then undid it which caused some prettier styling to take place, it has no functional difference.